google_guest_agent/network/manager.go

```diff
@@ -0,0 +1,39 @@
+package network
```

Comment:
we are missing the network package comment.

Reply:
Done.